third_party/forked/shell2junit/sh2ju.sh

```diff
@@ -137,11 +137,6 @@ function juLog() {
   content="$content
     <testcase assertions=\"1\" name=\"$name\" time=\"$time\" classname=\"$class\">
     $failure
-    <system-out>
```

Comment:
the output from some of the tests had non-printable characters that screwed up the junit results :(